sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/TypeUtils.scala

```diff
@@ -30,7 +30,7 @@ import org.apache.spark.sql.types._
 object TypeUtils extends QueryErrorsBase {
 
   def checkForOrderingExpr(dt: DataType, caller: String): TypeCheckResult = {
```

Comment:
can we check all the call sites of this function?

Reply:
Let's not touch it to reduce the blast radius.

Reply:
turns out we didn't need to change this at all, my bad 😄